deploy/docker/venice/keySchema.avsc

```diff
@@ -0,0 +1,11 @@
+{
+  "type": "record",
```

Comment:
We need to get `create table venice.foo` working :)

Reply:
Agreed, I will spend some time looking into this when I can. Should be a simple API call just as I'm doing to fetch schemas, just different than the current paradigm since it isn't managed via K8s.